src/Pickles/Pickles.DocumentationBuilders.Word/WordScenarioOutlineFormatter.cs

```diff
@@ -69,8 +71,18 @@ public void Format(Body body, ScenarioOutline scenarioOutline)
 
             foreach (var example in scenarioOutline.Examples)
             {
-                body.GenerateParagraph("Examples: " + example.Description, "Heading3");
-                this.wordTableFormatter.Format(body, example.TableArgument);
+                var paragraph = new Paragraph(new ParagraphProperties(new ParagraphStyleId { Val = "Heading3" }));
+                paragraph.Append( new Run( new RunProperties(), new Text( "Examples: " + example.Description ) ) );
```

Comment:
I like the visual style of the tags of a feature better. Could you adapt this code so it uses the same visual style as WordFeatureFormatter?